src/style.css

```
@@ -1 +1,97 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  /* list-style: none;
+  flex-direction: row; */
+  justify-content: space-around;
+  align-items: center;
+  width: 1200px;
+  height: 60px;
+  background: rgba(255, 255, 255, 1);
+}
+
+.header__header {
+  display: flex;
+  /* list-style: none;
+  flex-direction: row; */
+  justify-content: space-around;
+  align-items: center;
+  margin: 40px 0;
+  width: 1024px;
+  height: 60px;
+  background: rgba(255, 255, 255, 1);
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  text-transform: uppercase;
```

```suggestion
  text-transform: uppercase;
```
Це потрібно задавати для лінки